src/components/ArticleCard.jsx

```diff
@@ -18,6 +21,7 @@ const ArticleCard = ({
         <img src={article_img_url}></img>
         <p>votes: {votes}</p>
         <p>comment_count: {comment_count}</p>
+        <Link to={`/articles/${article_id}`}><button>Read</button></Link>
```

Comment:
I know you'll probably change the CSS later on, but might want to make this button a bit bigger and obvious from a UI point of view 👍